elasticsearch/sgconfig/sg_roles.yml

```diff
@@ -49,6 +49,48 @@ sg_role_admin:
   cluster:
     - CLUSTER_ALL
 
+sg_role_jaeger:
+  cluster:
+    - indices:data/write/bulk
+    - CLUSTER_MONITOR
+  indices:
+    '*jaeger-span-*-*-*':
+      '*':
+        - CRUD
+        - CREATE_INDEX
+        - READ
+        - MANAGE
+    '*jaeger-service-*-*-*':
+      '*':
+        - CRUD
+        - CREATE_INDEX
+        - READ
+        - MANAGE
+    '*jaeger-span-archive':
+      '*':
+        - CRUD
+        - CREATE_INDEX
+        - READ
+        - MANAGE
+    '*jaeger-span-archive-*':
+      '*':
+        - CRUD
+        - CREATE_INDEX
+        - READ
+        - MANAGE
+    '*jaeger-span-*':
```

Comment:
@ewolinetz I have also added these last two which are used for rollover. Are they ok?